pbspro.spec.in

```diff
@@ -513,6 +513,7 @@ fi
 %exclude %{pbs_prefix}/lib*/python/pbs_bootcheck*
 %exclude %{pbs_prefix}/libexec/install_db
 %exclude %{pbs_prefix}/libexec/pbs_habitat
+%exclude %{pbs_prefix}/libexec/pbs_schema_upgrade
```

Comment:
Yeah... this change is needed because when merged a PR of upgrading the DB schema, this line was added to pbspro.spec but it was missing from pbspro.spec.in.